.github/workflows/sandbox-deletion.yml

```
@@ -0,0 +1,41 @@
+
+jobs:
+  trigger-pipeline:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Check if PR was merged
+        id: check-merge
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:2:25: Double quote to prevent globbing and word splitting

(shellcheck)